Classes/common/CDTIncrementalStore.m

```diff
@@ -2208,6 +2221,75 @@ - (id)executeSaveRequest:(NSSaveChangesRequest *)saveRequest
     return @[];
 }
 
+- (id)executeBatchUpdateRequest:(NSBatchUpdateRequest *)updateRequest
+                    withContext:(NSManagedObjectContext *)context
+                          error:(NSError **)error
+{
+    NSError *err = nil;
+
+    NSFetchRequest *fetchRequest = [[NSFetchRequest alloc] init];
```

Comment:
ACK.  I like the "factory" methods better too ... don't know why I didn't use one here.